waspc/src/Analyzer/Evaluator/Combinators.hs

```diff
@@ -0,0 +1,175 @@
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- | This module contains combinators for building evaluators to convert
+-- typed Wasp AST into Haskell values.
```

Comment:
```suggestion
-- TypeChecker.AST into Wasp AST.
```
I believe I am now changing the change I suggested previously in this PR? But I feel I have better understanding now of what is happening and this sounds better.

Reply:
Obsolete